gerrit/README.md

```diff
@@ -70,11 +70,17 @@ The given revision is updated with the given message and/or label(s).
   just the resource name.
 
 * `message`: A message to be posted as a comment on the given revision.
+  The message can contain build metadata variables. (e.g.: $BUILD_ID)
+  See the [Concourse.CI Metadata Documentation](https://concourse.ci/implementing-resources.html#section_resource-metadata
+  for a complete list of variables.
 
 * `message_file`: Path to a file containing a message to be posted as a comment
   on the given revision. This overrides `message` *unless* reading
   `message_file` fails, in which case `message` is used instead. If reading
   `message_file` fails and `message` is not specified then the `put` will fail.
+  The message can contain build metadata variables. (e.g.: $BUILD_ID)
+  See the [Concourse.CI Metadata Documentation](https://concourse.ci/implementing-resources.html#section_resource-metadata
+  for a complete list of variables.
```

Comment:
I wonder if it makes sense to not do the interpolation for message_file since the program generating the file will have access to the env vars. Probably not a big deal either way.